packages/components/src/TableToolsTable/TableToolsTable.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
```

Comment:
Can we avoid the render of `input` container when there are no filters passed? By the `propTypes` spec, only `columns` and `items` required. 

![image](https://user-images.githubusercontent.com/31385370/180824675-a47727da-639b-42e9-8f62-b5f3e4db5ca4.png)

Reply:
Nice catch! Fixed.